src/main/java/io/a2a/spec/UnsupportedOperationError.java

```diff
@@ -0,0 +1,8 @@
+package io.a2a.spec;
+
+public class UnsupportedOperationError extends JSONRPCError {
+
+    public UnsupportedOperationError(int code, String message, Object data) {
+        super(code, message, data);
```

Comment:
Looks like the spec specifies default values for code and message:

https://github.com/google/A2A/blob/main/specification/json/a2a.json#L1450
https://github.com/google/A2A/blob/main/samples/python/common/types.py#L303-L304